tasks/Dockerfile

```diff
@@ -68,26 +64,24 @@ RUN groupadd -g 1111 -r user && useradd -r -g user -u 1111 user && \
     ln -snf /secrets/lorax-test-env.sh /home/user/.config/lorax-test-env && \
     ln -snf /run/secrets/webhook/.config--github-token /home/user/.config/github-token && \
     chmod g=u /etc/passwd && \
-    chmod -R ugo+w /cache /build /secrets /cache /home/user && \
+    chmod -R ugo+w /cache /secrets /cache /home/user && \
     ln -s /app/bin/firefox /usr/bin/firefox
 
 # Prevent us from screwing around with KVM settings in the container
 RUN touch /etc/modprobe.d/kvm-amd.conf && touch /etc/modprobe.d/kvm-intel.conf
 
 ENV LANG=C.UTF-8 \
     LIBGUESTFS_BACKEND=direct \
-    XDG_CACHE_HOME=/build \
```

Comment:
Hmm, this works, but it's a bit odd -- now all the builds literally happen in ~user/.cache/cockpit-project/.. , which is hard to see. I'll think about  how to move them to ~user cleanly.

Reply:
Fixed in another commit.